source/apps/change-deployment-models.txt

```diff
@@ -49,6 +50,18 @@ is in progress, the new migration does not run and fails with an error.
 Before You Begin
 ~~~~~~~~~~~~~~~~
 
+Before changing deployment models, please note the following: 
+
+- Suspended Triggers will restart.
+- Any existing Private Endpoints will need to be recreated for the new region.
+- The oplog replication window associated with any running Triggers is large 
```

Comment:
I don't really understand why this is included as a bullet. And is it associated with the first bullet? i.e. is a "suspended trigger" one that is restarted after being paused? 
is there a better way to phrase this so it's more obvious why I should note this?

Reply:
```suggestion
- Triggers and Device Sync operations will be paused during the migration.
```
(again, is a paused trigger == suspended trigger?)